projects/hslayers/src/components/query/feature-popup.component.ts

```diff
@@ -109,7 +109,11 @@ export class HsQueryFeaturePopupComponent implements OnDestroy {
     }
     if (getFeatures(feature)) {
       return (
-        'Cluster containing ' + getFeatures(feature).length + ' ' + 'features'
+        this.HsLanguageService.getTranslation('QUERY.clusterContaining') +
```

Comment:
I think its better to use interpolated params here and just have one string to translate in the json instead of 2 seperate.

```this.HsLanguageService.getTranslation(
                 'QUERY.clusterContaining',
                  {count: getFeatures(feature).length}
```

Reply:
Yes, I forgot about those!